src/pages/components/DateSelector.tsx

```diff
@@ -20,7 +21,7 @@ export function DateSelector({ time, onChange, customLabel, minDate }: DataAndTi
         return ''
       }
     }
-  }, [error])
+  }, [error, t])
```

Comment:
Maybe it would be better if we'll just remove the `useMemo` hook

Reply:
@NoamGaash 
It's a good idea to remove it.
while removing it I had thought about 2 how to implement it. 

1.  Just removing the useMemo and use the function.
2.  Take the function out of the component( and remove the useTranslation also) - will look better. 
does it matter which approach I go forward with? I did both of them and it works.

Reply:
That's up to you
I'd like to make a third suggestion - 
1. the function could return the translation key
2. the function can be inside the scope of the component, or underneath